### PR TITLE
Added additional resources on introduction to flexbox curriculum in foundation course

### PR DESCRIPTION
**Pull request by Sajwan055**
Here I have added ;
- I have added additional resource to the introduction to flexbox lesson in foundation course.
- It will help beginner to understand topic in better way.

### CODE REVIEW

**Review by fortypercenttitanium — CHANGES_REQUESTED** [review]
This looks like a fantastic resource. However, I noticed you linked only to one of the tutorials on flexbox.

In order to merge this, let's rename the link text to say "This flexbox tutorial..." as opposed to just giving the name of the site.

File: foundations/html_css/flexbox/flexbox-intro.md [hunk 1]
```diff
@@ -73,4 +73,5 @@ This section contains questions for you to check your understanding of this less
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-*   It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
\ No newline at end of file
+* [interneting is hard](https://www.internetingishard.com/html-and-css/flexbox/) is a
```
Comment (zachmmeyer) [inline foundations/html_css/flexbox/flexbox-intro.md:76]:
```suggestion
* [This Flexbox tutorial](https://www.internetingishard.com/html-and-css/flexbox/) is a
```

**Review by zachmmeyer — CHANGES_REQUESTED** [review]
Fix link text

**Review by zachmmeyer — APPROVED** [review]
Thanks for adding this additional resource

**Comment by Sajwan055** [comment]
Thank you so much for merging.

**Comment by sajwan05** [comment]
I am the ghost here sajwan055 -> sajwan05.

**Pull request merged**